Anatomy of a Device.ipynb

```diff
@@ -785,11 +785,10 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-     "tags" [
```

Comment:
This is missing a colon after `"tags"` which makes it invalid JSON. Probably my fault. With that fixed, I think the notebook builds without errors.

Reply:
I had problems with the documentation. just checking now, before MX meeting

Reply:
yeah, looks like I complicated it by moving the raises-exception tag. all fixed. will make sure it is correct for other notebooks as well